timesketch/frontend-ng/src/components/Scenarios/QuestionConclusion.vue

```diff
@@ -0,0 +1,123 @@
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <div>
+    <v-progress-linear v-if="loading" color="primary" indeterminate height="2"></v-progress-linear>
+    <v-list three-line>
+      <v-list-item @mouseover="showControls = true" @mouseleave="showControls = false">
+        <v-list-item-content>
+          <v-list-item-title style="font-size: 0.9em">
+            <strong>{{ conclusion.user.username }}</strong>
+          </v-list-item-title>
+          <v-list-item-subtitle>
+            <small>{{ conclusion.created_at | shortDateTime }} ({{ conclusion.created_at | timeSince }})</small>
+          </v-list-item-subtitle>
+
+          <div v-if="editable">
+            <v-textarea
+              style="font-size: 0.9em"
+              hide-details
+              auto-grow
+              outlined
+              v-model="conclusionText"
+              :value="conclusion.conclusion"
+            >
+            </v-textarea>
+            <v-card-actions>
+              <v-spacer></v-spacer>
+              <v-btn small text @click="editable = false"> Cancel </v-btn>
+              <v-btn small text color="primary" @click="saveConclusion()"> Save </v-btn>
+            </v-card-actions>
+          </div>
+          <div v-else style="max-width: 90%; font-size: 0.9em">{{ conclusion.conclusion }}</div>
+        </v-list-item-content>
+
+        <v-list-item-action
+          v-if="showControls && currentUser == conclusion.user.username"
+          style="position: absolute; right: 0"
+        >
+          <v-chip outlined style="margin-right: 10px">
+            <v-btn icon small>
+              <v-icon small @click="editConclusion()">mdi-square-edit-outline</v-icon>
+            </v-btn>
+            <v-btn icon small>
+              <v-icon small @click="deleteConclusion()">mdi-trash-can-outline</v-icon>
+            </v-btn>
+          </v-chip>
+        </v-list-item-action>
+      </v-list-item>
+    </v-list>
+  </div>
+</template>
+
+<script>
+import ApiClient from '../../utils/RestApiClient'
+
+export default {
+  props: ['question', 'conclusion'],
+  data: function () {
+    return {
+      conclusionText: '',
+      editable: false,
+      showControls: false,
+      loading: false,
+    }
+  },
+  computed: {
+    sketch() {
+      return this.$store.state.sketch
+    },
+    currentUser() {
+      return this.$store.state.currentUser
+    },
+  },
+  methods: {
+    saveConclusion: function () {
+      this.loading = true
+      this.editable = false
+      this.showControls = false
+      this.conclusion.conclusion = ''
+      ApiClient.editQuestionConclusion(this.sketch.id, this.question.id, this.conclusion.id, this.conclusionText)
+        .then((response) => {
+          this.$store.dispatch('updateScenarios', this.sketch.id).then(() => {
+            this.loading = false
+          })
+        })
+        .catch((e) => {})
+    },
```

Comment:
In the context of [an earlier comment](https://github.com/google/timesketch/pull/2451#discussion_r1049616090) we can either check here if the `this.conclusionText` is empty and prevent empty text. Or we trigger the `deleteConclusion()` function and follow the interpretation of the users intention if the variable is empty?
WDYT?

Reply:
Adding a check